### PR TITLE
Ob 4625 fix CVE

### PR DESCRIPTION
**Pull request by patric-dosch-vi**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by vi-sudo** [comment]
Task linked: [OB-4625 Tsys: Fix new CVEs that have been detected recently for keycloak image](https://app.clickup.com/t/6656196/OB-4625)

File: .github/workflows/dockerImage.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Setup JVM
         uses: actions/setup-java@v1
         with:
-          java-version: 11.0.10
+          java-version: 17
```
Comment (patric-dosch-vi) [inline .github/workflows/dockerImage.yml:30]: hey @tkuzynow, would you use a specific java version?

File: pom.xml [hunk 2]
```diff
@@ -21,11 +21,23 @@
       <artifactId>openapi-generator-maven-plugin</artifactId>
       <version>6.2.0</version>
       <scope>provided</scope>
+      <exclusions>
```
Comment (patric-dosch-vi) [inline pom.xml:24]: I would re-exclude them. I can't remember what you said about your first things to fix this issue.

File: .github/workflows/dockerImage.yml (hunk 1 above)
Comment (tkuzynow) [reply .github/workflows/dockerImage.yml:30]: no, I think it's fine as you have done it

File: pom.xml (hunk 2 above)
Comment (tkuzynow) [reply pom.xml:24]: OK, we can reexclude

**Review by tkuzynow — APPROVED** [review]

**Pull request merged**